### ISSUE TITLE
WARN: DEPRECATION: The 'default' option for attributes is being replaced by 'value' - please use it instead. attribute name: 'Inspec::Input'

### ISSUE THREAD

**Issue by IhorHDD** [dev-sec/cis-dil-benchmark#69]
Hello!

We are using your profile in inspec.yml dependency:
depends:
  - name: cis-dil-benchmark
    git: https://github.com/dev-sec/cis-dil-benchmark
    tag: '0.3.0'

**Describe the bug**
During uploading a profile to Chef-Automate I get an error below:
`WARN: DEPRECATION: The 'default' option for attributes is being replaced by 'value' - please use it instead. attribute name: 'Inspec::Input'`

I investigated a bit this error and tried next: I've excluded your profile from dependency and after that warning not appeared. After that, I included your profile to dependency and warning back.

I compared your profile with https://github.com/dev-sec/linux-baseline and found one difference, that can cause this warning. You have inspec.yml file, which include default:2. I can guess that default: 2 should be replaced with value: 2 for solving this issue.

```
attributes:
  - name: cis_level
    required: false
    description: 'CIS profile level to audit'
    default: 2
    type: numeric
```

**Expected behavior**
A warning "WARN: DEPRECATION: The 'default' option for attributes is being replaced by 'value' - please use it instead. attribute name: 'Inspec::Input'" shouldn;t appears on the uploading inspec profile to Chef-Automate step.

**Actual behavior**
"WARN: DEPRECATION: The 'default' option for attributes is being replaced by 'value' - please use it instead. attribute name: 'Inspec::Input'"

**Inspec Version**
 4.17.11 

If additional information required - please let me know.

Regards,
Ihor

**Comment by chris-rock** [comment]
@IhorHDD Thank you for reporting this issue. Until the CINC auditor https://cinc-project.gitlab.io is available, we need to be InSpec 3 compatible since Chef's new licensing terms prevents the dev-sec team from using InSpec 4 in our CI pipeline.

I think this specific change also works with the latest version of InSpec 3, therefore please go ahead and contribute a PR to remove the `default` deprecations.

**Comment by chris-rock** [comment]
Fixed in #70
